CMakeLists.txt

```diff
@@ -192,8 +192,8 @@ target_link_libraries(ni_grpc_device_server
 
 if(WIN32)
   target_link_libraries(ni_grpc_device_server Delayimp nidmm niScope niswitch nisync)
-  set_target_properties(ni_grpc_device_server PROPERTIES LINK_FLAGS "/DELAYLOAD:nidmm.dll /DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
-  set(ni_grpc_device_server CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /DELAYLOAD:nidmm.dll /DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
+  set_target_properties(ni_grpc_device_server PROPERTIES LINK_FLAGS "/DELAYLOAD:nidmm_64.dll /DELAYLOAD:niScope.dll /DELAYLOAD:niswitch.dll /DELAYLOAD:nisync.dll")
```

Comment:
Are we not replacing the import library version of nidmm.lib with static library version of nidmm.lib?

Reply:
the static version for 64 bit is not present for nidmm. I searched all locations of lib, I only found one 32 bit static lib, all 64 bit are import